### PR TITLE
Fix the Event.for_repo_issues call.

### PR DESCRIPTION
**Pull request by yallop**
Before:
```ocaml
# Lwt_main.run (Github.Monad.run (Github.Stream.to_list (Github.Event.for_repo_issues ~user:"dsheets" ~repo:"ISO8601.ml"  ())));;
Bad response: Ag_oj_run.Error("Line 1:\nMissing record fields public, payload, repo, 0jic_type")
Exception:
Failure
 "Bad response: Ag_oj_run.Error(\"Line 1:\\nMissing record fields public, payload, repo, 0jic_type\")\n".
```
After:
```ocaml
# Lwt_main.run (Github.Monad.run (Github.Stream.to_list (Github.Event.for_repo_issues ~user:"dsheets" ~repo:"ISO8601.ml"  ())));;
- : Github_t.issues_event list =
[{Github_t.issues_event_event = `Referenced;
  issues_event_issue =
   {Github_t.issue_url =
     "https://api.github.com/repos/dsheets/ISO8601.ml/issues/1";
    issue_html_url = "https://github.com/dsheets/ISO8601.ml/pull/1";
    issue_number = 1; issue_state = `Closed;
    issue_title = "Add unix dependency to the META file.";
    issue_body =
[...]
```

### CODE REVIEW

**Comment by dsheets** [comment]
See #107 but I think we need to support both the original flavor and the issue event specific variety. 😕

**Comment by yallop** [comment]
I've pushed an update that leaves the types used in `jar/` as they were.

**Comment by yallop** [comment]
I've added another function to the interface, `Event.for_repo_issue`, which retrieves the events for a single issue (f67cb0f).

The types are very slightly different to `Event.for_repo_issues`, unfortunately.

File: lib/github.atd [hunk 1]
```diff
@@ -511,6 +511,43 @@ type gollum_event = {
...
+  event: repo_issues_action;
+  ?assignee: user_info option;
+  ?label: label option;
+} <ocaml field_prefix="issues_event_">
+
+type repo_issues_event = {
+  event: repo_issues_action;
+  issue: issue;
+  ?assignee: user_info option;
+  ?label: label option;
```
Comment (dsheets) [inline lib/github.atd]: I don't think you need these fields that are duplicates of those in `repo_issue_event`.

Comment (yallop) [inline lib/github.atd]: Thanks.  Fixed (3e7c08b).

**Comment by dsheets** [comment]
Rebase merged in #150.

**Pull request closed without merging**